docs/src/content/docs/gonx/migration.md

```diff
@@ -0,0 +1,278 @@
+
+**Solution**:
+
+1. Ensure `go.mod` files exist in your Go projects
+2. Clear Nx cache: `nx reset`
+3. Verify plugin configuration in `nx.json`
+
+### Issue: Build Failures
+
+**Problem**: Builds fail due to path changes.
+
+**Solution**:
+
+1. Update any hardcoded paths in your Go code
+2. Ensure imports use proper module paths
+3. Check that `go.mod` files have correct module names
+
+### Issue: Custom Executors Missing
+
+**Problem**: Custom nx-go executors are no longer available.
+
+**Solution**:
+
+1. Use GoNx's built-in executors
+2. Create custom executors if needed
+3. Use shell commands as a fallback
+
+### Issue: Project Graph Issues
+
+**Problem**: Project dependencies aren't detected correctly.
+
+**Solution**:
+
+1. Ensure Go imports are correctly structured
+2. Use proper module paths
+3. Clear cache and regenerate: `nx reset && nx graph`
+
+## Feature Mapping
+
+Here's how nx-go features map to GoNx:
+
+| nx-go Feature        | GoNx Equivalent       | Notes                     |
+| -------------------- | --------------------- | ------------------------- |
+| `@nx-go/nx-go:build` | Inferred `build` task | Automatic detection       |
+| `@nx-go/nx-go:test`  | Inferred `test` task  | Improved caching          |
+| `@nx-go/nx-go:lint`  | Inferred `lint` task  | Uses `go fmt`             |
+| `@nx-go/nx-go:serve` | Inferred `serve` task | Runs from project root    |
+| Manual project.json  | Inferred tasks        | Cleaner project structure |
+| Custom executors     | Built-in + extensible | Modern Nx patterns        |
+
+## Post-Migration Testing
+
+After migration, verify everything works:
+
+### 1. Project Detection
+
+```bash
+nx show projects
+```
+
+### 2. Task Execution
+
+```bash
+nx build my-go-app
+nx test my-go-lib
+nx lint my-go-app
+```
+
+### 3. Project Graph
+
+```bash
+nx graph
+```
+
+### 4. Caching
+
+```bash
+nx build my-go-app
+nx build my-go-app  # Should use cache
+```
+
+## Rollback Plan
+
+If you need to rollback:
+
+1. **Restore backup**: Use your workspace backup
+2. **Reinstall nx-go**: `npm install @nx-go/nx-go`
+3. **Restore configurations**: Restore `nx.json` and `project.json` files
+4. **Clear cache**: `nx reset`
+
+## Getting Help
+
+If you encounter issues during migration:
+
+1. **Check documentation**: Review GoNx documentation thoroughly
+2. **Search issues**: Look for similar issues on GitHub
+3. **Join Discord**: Get real-time help on our [Discord server](https://discord.gg/zjDCGpKP2S)
+4. **Create issue**: Report bugs or request help on GitHub
+
+## Benefits After Migration
+
+Once migrated, you'll enjoy:
+
+- **Better performance**: Improved caching and task execution
+- **Cleaner projects**: No more project.json clutter
+- **Modern features**: Latest Nx capabilities
+- **Better reliability**: Official Go command integration
+- **Active maintenance**: Regular updates and improvements
+
+The migration effort pays off with a more maintainable and performant setup! 🚀
```

Comment:
Suggestion: Less is more. Let's remove this section since we have mentioned most of this in multiple instances.